docs/pages/reference/cli.mdx

```diff
@@ -2078,7 +2078,7 @@ Configure the GitHub auth connector.
 Required params `--id` and `--secret` come from registering a GitHub OAuth app, which you can read about in the [GitHub documentation](https://docs.github.com/en/developers/apps/building-oauth-apps/creating-an-oauth-app).
 
 The flag `--teams-to-roles` can be provided multiple times to specify which GitHub Teams are assigned which roles.
-In this example, members of the `octocats` team in GitHub will be assigned the `devs`, `access`, and `editor` roles in Teleport.
+In this example, members of the `devs` team in the GitHub org `octocats` will be assigned the `access`, and `editor` roles in Teleport.
 
 ```code
 $ tctl sso test --id=GITHUB_CLIENT_ID --secret=GITHUB_SECRET --teams-to-roles=octocats,devs,access,editor [<other-flags>]
```

Comment:
I don't see a `--teams-to-roles` flag on the `tctl sso test` command. Something seems off here.

Reply:
Yea, this should probably be `tctl ssh configure github`. CC @Tener

Reply:
Yep, this should be `tctl sso configure github` as per section header.